compiler/rustc_middle/src/traits/select.rs

```diff
@@ -137,12 +137,6 @@ pub enum SelectionCandidate<'tcx> {
         is_const: bool,
     },
 
-    /// Builtin implementation of `DiscriminantKind`.
```

Comment:
Can you add a comment to `BuiltinCandidate` to give some context of the types of predicates it covers?